avro2py/codegen.py

```diff
@@ -363,10 +362,16 @@ def bridge_namespaces(from_namespace: List[str], to_namespace: List[str], target
         remaining_to_components = [c for c in paired_remaining_to_components if c is not None]
 
         if remaining_from_components:
+            level = len(remaining_from_components)
+            for namespace_name in self.namespaces.keys():
+                namespace_parts = namespace_name.split('.')
+                if namespace_parts[0:len(from_namespace)] == from_namespace and len(namespace_parts) > len(from_namespace):
+                    level += 1
```

Comment:
The `+= 1` here is strange to me, since we immediately break, and I think supports that this would be better expressed as some method like that's doing an `if cond: return True` rather than `if cond: x(); break`.

Reply:
no more += 1